shared/local-debug.native.js

```diff
@@ -67,7 +67,7 @@ export function initTabbedRouterState (state) {
     tabs: {
       ...state.tabs,
       [Tabs.loginTab]: createRouterState([], []),
-      [Tabs.moreTab]: createRouterState(['devMenu', 'dumbSheet'], [])
+      [Tabs.moreTab]: createRouterState(['devMenu', 'styleSheet'], [])
```

Comment:
I'm going to branch off this to do android. I'll remove it when it gets
collapsed down

On Wednesday, April 27, 2016, Marco Munizaga <notifications@github.com>
wrote:

> In shared/local-debug.native.js
> <https://github.com/keybase/client/pull/2719#discussion_r61345877>:
>
> > @@ -67,7 +67,7 @@ export function initTabbedRouterState (state) {
> >      tabs: {
> >        ...state.tabs,
> >        [Tabs.loginTab]: createRouterState([], []),
> > -      [Tabs.moreTab]: createRouterState(['devMenu', 'dumbSheet'], [])
> > +      [Tabs.moreTab]: createRouterState(['devMenu', 'styleSheet'], [])
>
> Did you mean to check this in?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/2719/files/ff0c487c5b305237a7ebdb3edc8dbdd4d47a44d2#r61345877>
>